.github/workflows/tuist.yml

```diff
@@ -86,35 +86,6 @@ jobs:
           path: |
             /tmp/tuist/**
 
-  cache-warm-silicon:
```

Comment:
I am removing this as in the new version of `tuist`, we will compile the binaries for all architectures, so we only need to run the cache warm once. Additionally, the current `tuist` version is broken when used on both archs for macOS binaries (iOS should be fine)